crates/ethcore/src/miner/stratum.rs

```diff
@@ -47,6 +47,14 @@ pub struct Options {
     pub secret: Option<H256>,
 }
 
+fn clean_0x(s: &str) -> &str {
```

Comment:
This function is added twice in the PR, here and in bin/oe/helpers, could it be moved to utils?

Reply:
It is a one `if` function, and I am fine with it for now.